src/store/auth/sagas.ts

```diff
@@ -184,6 +184,12 @@ export function* setUsageMode({
   payload,
 }: ReturnType<typeof setUsageModeRequest>) {
   const authApi: AuthApi = yield call(configureAuthSaga);
+
+  // passthrough mode can't be enabled until all users are deleted
+  if (payload === 'PASSTHROUGH') {
+    yield put(nukeUsersRequest());
```

Comment:
```suggestion
    yield put(nukeUsersRequest());
    yield take(getType(nukeUsersSuccess));
```

Gotta wait until users are actually deleted